coursedata/adventures/nb_NO.yaml

```diff
@@ -63,24 +63,24 @@ adventures:
     turtle:
         levels:
             4:
-                start_code: "{print} 'Drawing figures'\nangle {is} 90\n{turn} angle\n{forward} 25\n"
-                story_text: "## Let's draw\nIn level 4 you have to use quotation marks with `{print}` and `{ask}`. Also when drawing!\n"
-                example_code: "## Example Hedy code\n\n```\n{print} 'Drawing figures'\nangle {is} 90\n{turn} angle\n{forward} 25\n{turn} angle\n{forward} 25\n```\n"
+                start_code: "{print} Tegner figurer\nvinkel {is} 90\n{turn} vinkel\n{forward} 25\n"
```

Comment:
Hi @Hakon!

There is an error here, you removed the '' around Tegner figure, it should be: 

```
print 'Tegner figurer'
vinkel is 90
turn vinkel
forward 25
```

Probably easiest if you fix that in Weblate.

Reply:
Thank you! I will fix

Reply:
I think i have already gone back to fix that, could you verify for me?